core/src/main/java/com/google/googlejavaformat/java/StringWrapper.java

```diff
@@ -197,12 +197,10 @@ private void indentTextBlocks(
         ImmutableList<String> initialLines = text.lines().collect(toImmutableList());
         String stripped = stripIndent(initialLines.stream().skip(1).collect(joining(separator)));
         ImmutableList<String> lines = stripped.lines().collect(toImmutableList());
-        int deindent =
-            initialLines.get(1).stripTrailing().length() - lines.get(0).stripTrailing().length();
 
         int startColumn = lineMap.getColumnNumber(startPosition);
         String prefix =
-            (deindent == 0 || lines.stream().anyMatch(x -> x.length() + startColumn > columnLimit))
```

Comment:
We don't have to compute "deindent" at all here, I think, since no text block is allowed to appear at the top level of the syntax tree (?). Even if we wanted to allow it, the condition should be ```prefix = (startColumn == 0 ? "" : " ".repeat(startColumn -1)```.